jobs/kubelet-windows/templates/bin/drain.ps1.erb

```diff
@@ -0,0 +1,69 @@
+trap { $host.SetShouldExit(1) }
+
+$OutLog = "C:\var\vcap\sys\log\kubelet-windows\drain.stdout.log"
+$ErrLog = "C:\var\vcap\sys\log\kubelet-windows\drain.stderr.log"
+
+
+function main {
+  if (kubelet_is_running) {
+    retry "drain_kubelet" $function:drain_kubelet
+    # watch_disks TODO(BM/LH): implement this?
```

Comment:
Hey Folks, is this something you all planned to do? @benmoss

Reply:
We need to look into how persistent volumes work with Windows again to see if there's anything we need to do when draining